router/proxy/export.go

```diff
@@ -0,0 +1,3 @@
+package proxy
+
+var StickyCookie = stickyCookie
```

Comment:
yeah, it's on my TODO list to alter the tests so that this exported var/const is not necessary.